### PR TITLE
Fix Rancher cluster sync logger

### PR DESCRIPTION
**Pull request by fredtibbitts**
The logger in the Rancher cluster sync code was not being set correctly. As a result the log level configured on the VPO (via command line switch) was not honored. I tested by setting the log level to debug on the VPO and confirmed that debug log message appear.

I also changed a Progress message to a Debug message.

### CODE REVIEW

**Review by davisgomes — APPROVED** [review]

File: platform-operator/controllers/clusters/sync_rancher_clusters.go [hunk 1]
```diff
@@ -33,11 +32,15 @@ var clustersResponseHash []byte
...
 type RancherClusterSyncer struct {
 	client.Client
+	Log *zap.SugaredLogger
 }
 
 // StartSyncing starts the Rancher cluster synchronization loop
 func (r *RancherClusterSyncer) StartSyncing() {
-	log := r.initLogger()
+	// initialize logger - the AddCallerSkip is needed otherwise the caller in the log message always shows "vzlog.go"
+	l := r.Log.Desugar().WithOptions(zap.AddCallerSkip(2))
```
Comment (pfmackin) [inline platform-operator/controllers/clusters/sync_rancher_clusters.go:41]: why is this needed for Rancher?

**Review by desagar — APPROVED** [review]

File: platform-operator/controllers/clusters/sync_rancher_clusters.go (hunk 1 above)
Comment (fredtibbitts) [reply platform-operator/controllers/clusters/sync_rancher_clusters.go:41]: Per conversation, I don't understand why this is needed. Will file a ticket for additional investigation.
